src/components/Button/button.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import { expect } from 'vitest';
+
+import { Button } from '.';
+
+describe('Button tests', () => {
+  test('if primary button is rendered', () => {
+    render(<Button type="button" variant="primary" label="Primary" onClick={() => {}} />);
```

Comment:
- if you write const {getByText} = render(<Button type="button" variant="primary" label="Primary" onClick={() => {}} />) then we will have less code
- type="button"  label="Primary" onClick={() => {}} is default/basic props so you can hand only variant and the other props will have one declaration